packages/client/modules/meeting/components/TemplateScaleValueItem.tsx

```diff
@@ -74,13 +75,14 @@ const TemplateScaleValueItem = (props: Props) => {
     submitMutation()
     RemovePokerTemplateScaleValueMutation(atmosphere, {scaleId, label}, {}, onError, onCompleted)
   }
+  const isSpecial = isSpecialPokerLabel(label)
   return (
     <ScaleValueItem
-      ref={dragProvided.innerRef}
-      {...dragProvided.dragHandleProps}
-      {...dragProvided.draggableProps}
+      ref={dragProvided?.innerRef}
+      {...dragProvided?.dragHandleProps}
+      {...dragProvided?.draggableProps}
       isDragging={isDragging}
-      isHover={isHover}
+      isHover={!isSpecial && isHover}
```

Comment:
nittiest of nits: i'd be nice to pass `isSpecial` to the component & let it decide on the logic because `isHover = !isSpecial` could cause some confusion, but since it's all in the same file, i think we're OK here